freqtrade/exchange/exchange.py

```diff
@@ -831,6 +869,28 @@ def _get_params(self, ordertype: str, leverage: float, time_in_force: str = 'gtc
             params.update({param: time_in_force})
         return params
 
+    def _amount_to_contracts(self, pair: str, amount: float):
+
+        contract_size = None
+        if ('contractSize' in self.markets[pair]):
+            contract_size = self.markets[pair]['contractSize']
+
+        if (contract_size and self.trading_mode == TradingMode.FUTURES):
+            return amount / contract_size
+        else:
+            return amount
+
+    def _contracts_to_amount(self, pair: str, num_contracts: float):
+
+        contract_size = None
+        if ('contractSize' in self.markets[pair]):
+            contract_size = self.markets[pair]['contractSize']
+
+        if (contract_size and self.trading_mode == TradingMode.FUTURES):
+            return num_contracts * contract_size
+        else:
+            return num_contracts
+
```

Comment:
Please move these lines next to `_get_contract_size` (either this part up, or the other ones down).

That way, all methods that belong together are "somehow" grouped within the file - so we don't have to look all over the place for things that belong together.